### DISCUSSION TITLE
Plan ID for Fully Insured Business

### DISCUSSION THREAD

**Discussion by KMoriarity** [CMSgov/price-transparency-guide#548] [Q&A] [answered]
What plan ID should be reported for fully-insured group business that doesn't have HIOS ID.  

Should this this be health insurance issuer's EIN or the insured group employer's EIN?

**Comment by ghost** [comment]
A fully insured group would have the file generated with their EIN as they are the insurer even if they use an insure carrier to perform the claims adjudication for them.

**Reply by bryanbriegel** [reply]
Agree, the EIN of the employer that is sponsoring the fully insured plan.

**Reply by KMoriarity** [reply]
> A fully insured group would have the file generated with their EIN as they are the insurer even if they use an insure carrier to perform the claims adjudication for them.

I'm not sure I understand your comment that the group is the insurer.  The group doesn't carry any of the risk so they aren't acting as the insurer.  

@shaselton-usds  Can you clarify which EIN should be used?

**Reply by bryanbriegel** [reply]
The employer is sponsoring the plan, and are the group health plan sponsor.  They are not the issuer.  They are paying the issuer/insurer who is bearing the risk for the group health plan.

**Comment by shaselton-usds** [comment, marked as answer]
@KMoriarity [Pulling from the rule](https://www.federalregister.gov/d/2020-24591/p-582)

> The first content element that plans and issuers will be required to include in the machine-readable files is the name and identifier for each coverage option offered by a group health plan or health insurance issuer. For the identifier, the Departments proposed that plans and issuers use their Employer Identification Number (EIN) or Health Insurance Oversight System (HIOS) IDs, as applicable. The Departments sought comment on whether EINs and HIOS IDs are the appropriate identifiers for this purpose. The Departments also sought comment on whether there are other plan or issuer identifiers that should be considered and adopted.

It sounds like in your case, a fully-insured plan (opposed to a self-insured scenario) would be the insurer's information. Does this help?

**Reply by KMoriarity** [reply]
After internal discussion, I realize there is possibility that I didn't include in my original question.  

For fully insured business that doesn't have a 14 digit HIOS, is there a regulatory hierarchy or preference for deciding between the health insurance issuer's 5 digit HIOS and and issuer's EIN?  If there is any hierarchy or preference, is there any difference in that guidance between group and individual business?

@shaselton-usds, could you advise again?

**Reply by shaselton-usds** [reply]
@KMoriarity You can find the [hierarchy here](https://github.com/CMSgov/price-transparency-guide/pull/447#issuecomment-1102994675).

**Reply by KMoriarity** [reply]
Thank you for your quick response. I'm interpreting the linked documentation to favor the 5 digit HIOS over the issuer EIN, is that correct?

**Reply by shaselton-usds** [reply]
Correct.
